Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealitySpeechCommandsProfileInspector.cs

```diff
@@ -19,6 +18,19 @@ public class MixedRealitySpeechCommandsProfileInspector : MixedRealityBaseConfig
         private static readonly GUIContent KeyCodeContent = new GUIContent("KeyCode", "The keyboard key that will trigger the action.");
         private static readonly GUIContent ActionContent = new GUIContent("Action", "The action to trigger when a keyboard key is pressed or keyword is recognized.");
 
+        private static readonly GUIContent SpeechConfidenceContent = new GUIContent("Confidence Level", "The speech recognizer's minimum confidence level setting that will raise the action.");
+        private static readonly GUIContent[] SpeechConfidenceOptionContent =
```

Comment:
Most likely wont need this if confidence is changed to an enum instead of an int